examples/e-commerce/yarn.lock

```diff
@@ -4412,6 +4412,21 @@ instantsearch.js@3.5.3, instantsearch.js@^3.5.1:
     prop-types "^15.5.10"
     qs "^6.5.1"
 
+instantsearch.js@^3.6.0:
```

Comment:
same issue as with the helper earlier, only one version should be here

Reply:
I wpdated instantsearch.js version in the example to avoid this https://github.com/algolia/vue-instantsearch/pull/732/commits/5805fa536fb59017b2ae88fb6e1d11c160f2b922